interpreter/cling/lib/Interpreter/NullDerefProtectionTransformer.cpp

```diff
@@ -203,24 +171,31 @@ namespace cling {
           if (ArgIndexs.test(index)) {
             // Get the argument with the nonnull attribute.
             Expr* Arg = CE->getArg(index);
+            NodeContext newArg = SynthesizeCheck(Arg->getLocStart(), Arg);
+            if (newArg.isSingleStmt()) {
+              assert ((clang::Expr*)newArg.getStmt()
```

Comment:
See `isa` comment above. Actually - given that this shows up three times - can we just let `SynthesizeCheck` return an `Expr*`?